### PR TITLE
TW-2099: Design review v2.6.7

### PR DESCRIPTION
**Pull request by nqhhdev**
## Ticket
- #2099 

### Resolved:

- [x] Menu background

Screenshot 2024-10-23 at 10 10 09
Screenshot 2024-10-23 at 10 10 16
Screenshot 2024-10-23 at 10 10 29
Screenshot 2024-10-23 at 10 10 44
Screenshot 2024-10-23 at 10 11 14

- [x] Padding for chat list item
Screenshot 2024-10-22 at 15 44 14
Screenshot 2024-10-22 at 15 52 54

- [x] Update padding in chat

Screenshot 2024-10-24 at 01 14 18
Screenshot 2024-10-24 at 01 33 22

### CODE REVIEW

**Comment by hoangdat** [comment]
- [ ] should align also the composer

image

**Comment by hoangdat** [comment]
- [ ] align message bubble of Direct chat 

Screenshot 2024-10-24 at 10 03 18

**Comment by hoangdat** [comment]
- [ ] back button in web still not correct in web app
Screenshot 2024-10-25 at 15 11 16
Screenshot 2024-10-25 at 15 11 06
Screenshot 2024-10-25 at 15 10 56

**Comment by nqhhdev** [comment]
hi @hoangdat , I fixed all the comments and pushed it. pls help me review again

File: ios/Podfile [hunk 1]
```diff
@@ -33,9 +33,9 @@ target 'Runner' do
 
   flutter_install_all_ios_pods File.dirname(File.realpath(__FILE__))
   
-  target 'Twake Share' do
-    inherit! :search_paths
-  end
+#   target 'Twake Share' do
+#     inherit! :search_paths
+#   end
```
Comment (hoangdat) [inline ios/Podfile:38]: why we need it?

File: ios/Runner.xcodeproj/project.pbxproj
Comment (hoangdat) [inline ios/Runner.xcodeproj/project.pbxproj:1]: idem

File: ios/Podfile (hunk 1 above)
Comment (nqhhdev) [reply ios/Podfile]: reverted

File: ios/Runner.xcodeproj/project.pbxproj
Comment (nqhhdev) [reply ios/Runner.xcodeproj/project.pbxproj:1]: reverted

**Review by hoangdat — APPROVED** [review]

**Pull request merged**